online-shopping-website/backend/tests/verifyBrandCreation.test.ts

```diff
@@ -0,0 +1,35 @@
+import hasRequiredBrandCreationParams from "../src/helpers/verifyBrandCreation";
+
+const data = ["name", "description"];
+let requestBody: any = { name: 'name', description: 'description'};
+
+describe("Required Brand Creation Params Tests", () => {
+    test("No name", () => {
+        requestBody.name = undefined;
+        requestBody.description = data[1];
+
+        expect(hasRequiredBrandCreationParams(requestBody)).toEqual(false);
+    });
+
+    test("No description", () => {
```

Comment:
Maybe change the test name to "No description, still valid" or something
Just so it's more obvious that while there's description, the brand can still be created!

Reply:
done!! thanks :-))